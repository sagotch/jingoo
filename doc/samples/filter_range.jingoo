<dl>
  <dt>range(0,3)</dt>
  <dd>{% for x in range(0,3) %}{{x}}{% endfor %}</dd>
  <dt>range(3,0)</dt>
  <dd>{% for x in range(3,0) %}{{x}}{% endfor %}</dd>
  <dt>range("a","e")</dt>
  <dd>{% for x in range("a","e") %}{{x}}{% endfor %}</dd>
</dl>
