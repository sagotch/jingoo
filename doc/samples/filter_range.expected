<dl>
  <dt>range(0,3)</dt>
  <dd>0123</dd>
  <dt>range(3,0)</dt>
  <dd>3210</dd>
  <dt>range("a","e")</dt>
  <dd>abcde</dd>
</dl>
